chromeapp/test/scripts/dnssd/dns-util-test.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
```

Comment:
I opened Issue #14 for unicode.

I also added tests for the short and long extremes of valid URLs, as well as asserted that all but a sanity check `throw` statement does indeed throw.